src/python/pants/jvm/resolve/coursier_fetch.py

```diff
@@ -521,14 +521,15 @@ def classpath_arg(self, root: Optional[str] = None) -> str:
             directory for the process input `Digest`.
         """
 
-        def maybe_add_prefix(file_name: str) -> str:
+        return ":".join(self.reified_filenames(root))
+
+    def reified_filenames(self, root: Optional[str] = None) -> Iterable[str]:
```

Comment:
I added the moral equivalent of this method in #13061, but you shouldn't need to interact with this class anymore: rather, the `Classpath` type that it adds.